pkg/job/job.go

```diff
@@ -0,0 +1,28 @@
+package job
+
+import (
+	"github.com/goharbor/harbor-scanner-clair/pkg/harbor"
+)
+
+type Status int
+
+const (
+	Queued Status = iota
+	Pending
+	Finished
+	Failed
```

Comment:
Suggest to define these statuses: `Pending`, `Running`, `Finished` and `Failed`.

When push job to pool its status is `Pending`, when pool get job and execute its status changed to `Running`.

Reply:
Agree. Changed.